metriq/models/__init__.py

```diff
@@ -52,7 +52,7 @@
     MethodCreateRequest,
     MethodUpdateRequest,
 )
-from metriq.models.moderationReport import (
+from metriq.models.moderation_report import (
```

Comment:
This model cannot be found when installing the package, I guess the old file has been removed in this PR but not replaced

Reply:
Thanks to your PR, I think this works now. Thank you again, @karlaspuldaro !